store/sql_store.go

```diff
@@ -110,6 +111,14 @@ func initConnection() *SqlStore {
 		}
 	}
 
+	if len(utils.Cfg.SqlSettings.DataSourceSearchReplica) == 0 {
+		sqlStore.searchReplica = sqlStore.master
```

Comment:
Make sure when it's blank it doesn't go to a single RR as well. It goes though the regular system.